modules/testing/src/views/full/Scenario.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 
 import { Panel, Label, Glyphicon } from 'react-bootstrap'
-import { MdExpandLess, MdExpandMore } from 'react-icons/md'
+import { MdExpandLess, MdExpandMore, MdDelete } from 'react-icons/md'
```

Comment:
We are trying to get rid of bootstrap and react-icons whenever possible, can you use Blueprint's icon s instead ?

Reply:
sure thing !